tools/mothballer.py

```diff
@@ -161,8 +162,8 @@ def package(categoryname, categorydir, seed):
     zf.close()
     zfraw.seek(0)
     return zfraw
```

Comment:
shouldn't this be `zipfileraw`?

Reply:
`build_category` uses zipfileraw as a `NamedTemporaryFile`, to which it write the result of the `package` function, which uses `zfraw` to represent a file-like `BytesIO` object that has zip-formatted contents, which are generated by `zf`

Reply:
Okay that is a bad choice of variable names but you don't have to fix it in your merge.